doc/source/examining/loading_data.rst

```diff
@@ -271,16 +271,18 @@ will be in code units.
 
 .. _loading-orion-data:
 
-BoxLib Data
+AMReX / BoxLib Data
 -----------
 
-yt has been tested with BoxLib data generated by Orion, Nyx, Maestro, Castro, 
-and WarpX. Currently it is cared for by a combination of Andrew Myers, Chris
-Malone, Matthew Turk, and Mike Zingale.
+AMReX and BoxLib share a frontend (currently named `boxlib`), since
+the file format is identical.  yt has been tested with AMReX/BoxLib
+data generated by Orion, Nyx, Maestro, Castro, IAMR, and
```

Comment:
just today actually.  Brian has been working on it.  Latest yt works with it out of the box